src/footprints/choc_hotswap.js

```diff
@@ -1,10 +1,21 @@
 module.exports = {
```

Comment:
I know this is not necessarily related to this particular PR, but I think we could/should merge hotswap footprint variants with their vanilla ones, and make this a parameter instead (default = non-hotswap). Let's return to this...